### ISSUE TITLE
Question: separate repo for l10n

### ISSUE THREAD

**Issue by sonnyp** [dialect-app/po#19]
Hello :wave: 

I've been struggling with l10n for my 6+ apps. I don't want git l10n commit messages for every single contribution.
I setup a separate branch in the repos where Weblate pushes the l10n updates. On release I would squash merge it into `main`.

 It worked fine for a while but I endup with conflicts and I don't have time to solve them.
 
 How is your experience with using a separate repo for translations? Could you share your setup / weblate settings ?
 
 I'd be interested in documenting this publicly if it works well.

**Comment by rafaelmardojai** [comment]
In general, I don't mind about commits per translation updates for my projects, but we agreed to separate them for Dialect, to have a cleaner git history and for our automation regarding language name translations.

I will say that the experience is good, we usually only touch this repo when updating strings from the UI.

The setup is not really special, just a git submodule and a regular weblate settings. I don't know how to exactly share the settings, screenshots will work? 😅

One important note is not to forget to tag releases in this repo as well, to make life easier for packagers.

**Comment by rafaelmardojai** [comment]
> Is the `pot` file in `po/ui` generated by Weblate?

Nope, it is manually generated by us with meson/ninja.

**Comment by rafaelmardojai** [comment]
These are de add-ons we have enabled:
![Captura de pantalla 2023-09-24 a las 10 44 31](https://github.com/dialect-app/po/assets/6210397/6f590d28-596f-4d1e-8b9f-e8a9831ff345)

Settings:
![Screenshot 2023-09-24 at 10-49-13 Dialect_Dialect](https://github.com/dialect-app/po/assets/6210397/40d199bb-a015-4a7a-a567-47379479dc93)
![Screenshot 2023-09-24 at 10-49-22 Dialect_Dialect](https://github.com/dialect-app/po/assets/6210397/7f5cb1c1-ea50-49c0-a867-ba5f1866557b)
![Screenshot 2023-09-24 at 10-49-41 Dialect_Dialect](https://github.com/dialect-app/po/assets/6210397/cb097613-7107-418b-8e84-0b205d7055a0)

**Comment by rafaelmardojai** [comment]
We also have the Weblate's webhook configured in Github

**Comment by sabriunal** [comment]
When I compare UI and strings in Weblate instence I see there are several (15~) missing strings in Weblate instence. 

![Ekran Görüntüsü - 2023-12-31 16-39-00](https://github.com/dialect-app/po/assets/6218679/c9f138f6-084d-4112-b56a-319a76cea182)
![Ekran Görüntüsü - 2023-12-31 16-38-44](https://github.com/dialect-app/po/assets/6218679/f5d13160-3b4c-41b2-88e1-29dcf85ead89)

**Comment by rafaelmardojai** [comment]
@sabriunal Yes, we have merged a few contributions that added new strings recently. We sync po files to code changes manually, and we haven't done it yet.
